pkg/metrics/otel.go

```diff
@@ -145,16 +146,9 @@ processors:
           - git_sync_depth_override_count_total
           - no_ssl_verify_count_total
           - kcc_resource_count
-      exclude:
-        match_type: strict
-        metric_names:
-          - rg_reconcile_duration_seconds
-          # TODO remove kcc_resource_count_total rule once Resource Group Controller
-          # 1.0.9 is updated into Config Sync. This metric was unintentionally
-          # included by the 'regex include' filter above and is not included in
-          # our Monarch metric definitions
-          - kcc_resource_count_total
   # Remove custom configsync metric labels that are not registered with Monarch
+  # This action applies to all metrics that are sent through the pipeline that
```

Comment:
Did you test this locally to make sure that it does not cause any formatting issue (https://github.com/GoogleContainerTools/kpt-config-sync/commit/9ebabf3b4723a90b67c0ade9fa36b64fa380db45)?

Reply:
Yeah good point, I made sure there is no trailing spaces but will test to check the format

Reply:
Tested, format is without issue.